cmd/boost/dummydeal.go

```diff
@@ -21,14 +22,17 @@ var dummydealCmd = &cli.Command{
 
 		ctx := lcli.DaemonContext(cctx)
 
-		log.Debug("Get boost identity")
-
-		res, err := boostApi.ID(ctx)
+		log.Debug("Make API call to start dummy deal")
+		rej, err := boostApi.MarketDummyDeal(ctx)
 		if err != nil {
-			return xerrors.Errorf("couldnt get boost identity: %w", err)
+			return xerrors.Errorf("creating dummy deal: %w", err)
 		}
 
-		spew.Dump(res)
+		if rej != nil && rej.Reason != "" {
+			fmt.Printf("Dummy deal rejected: %s\n", rej.Reason)
```

Comment:
Same comment about `fmt.Print` and `log` here, but at this specific place, I don't care so much, because this will likely change in the coming days when we make more progress, so fine to leave as is.